docs/zh/latest/admin-api.md

```diff
@@ -1319,6 +1319,14 @@ Content-Type: text/plain
 
 Plugin 资源请求地址：/apisix/admin/plugins/{plugin_name}
 
+### 请求参数
+
+| 名称 | 描述 | 默认 |
+| --------- | -------------------------------------- | -------- |
+| subsystem | 插件子系统。 | http |
+
+可以在子系统上过滤插件，以便在通过查询参数传递的子系统中搜索 ({plugin_name})。请注意，此查询参数在“/apisix/admin/plugins/list”上不可用
```

Comment:
I think it's supported on /apisix/admin/plugins/list?

Reply:
It wasn't present present previously and was later added in the pull request. I removed the later comment.